docs-chef-io/content/inspec/resources/k8sobject.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "k8sobject resource"
+draft = false
+gh_repo = "inspec"
+platform = "k8s"
+
+[menu]
+[menu.inspec]
+title = "k8sobject"
+identifier = "inspec/resources/k8s/K8s Object"
+parent = "inspec/resources/k8s"
++++
+
+
+Use the `k8sobject` Chef InSpec audit resource to test the configuration of...
+
+## Installation
+
+## Syntax
+
+```ruby
+describe k8sobject(type: K8s_RESOURCE_TYPE, namespace: NAMESPACE, name: RESOURCE_NAME) do
+  #...
+end
+```
+
+## Parameters
+
+`type`
+: type of the K8s resource that is for query
```

Comment:
@sathish-progress - this sentence look like a fragment. 

Can we word it like this -  type of the K8s resource that is used for quering ...?